code/__DEFINES/metacoin.dm

```diff
@@ -1,14 +1,14 @@
 /// Rewarded when you complete all your objectives as a traitor
 #define METACOIN_GREENTEXT_REWARD        200
```

Comment:
I would actually like a reduction in greentext coins, rather than a massive increase of other coins.